cedar-policy/CHANGELOG.md

```diff
@@ -12,15 +12,14 @@ Starting with version 3.2.4, changes marked with a star (*) are _language breaki
 ## [Unreleased]
 Cedar Language Version: TBD
 
+## [4.1.0] - TBD
```

Comment:
Will also need to define the `[4.1.0]` link at the end of the file.

Reply:
Got it.